tests/process.rs

```diff
@@ -34,3 +33,33 @@ fn test_process_refresh() {
         true
     );
 }
+
+#[test]
+#[cfg(windows)]
+fn test_get_cmd_line() {
+    let p = std::process::Command::new("ping")
+        .arg("localhost")
+        .arg("-n")
+        .arg("2")
+        .spawn()
+        .unwrap();
+    let mut s = sysinfo::System::new();
+    s.refresh_processes();
+    let process = s.get_process(p.id() as sysinfo::Pid).unwrap();
+    assert_eq!(process.cmd(), &["ping", "localhost", "-n", "2"]);
+}
+
+#[test]
+#[cfg(unix)]
```

Comment:
I'd prefer `#[cfg(not(windows))]`if you don't mind here. :)

Reply:
Yes, it is better. I don't have macOS and Raspberry and can't test the code on these platforms. If you are sure what it is works, you can accept it - I fixed it.

Reply:
Well, the CI is there for that part as well. ;)